core/models/model.py

```diff
@@ -262,6 +263,26 @@ class DrfModelSerializer(serializers.ModelSerializer):
 
     model = serializers.SerializerMethodField()
 
+    def __init__(self, *args, **kwargs):
+        # Don't pass the 'fields' arg up to the superclass
+        fields = kwargs.pop('fields', None)
```

Comment:
Nit: Can we rename `fields` arg to `requested_fields` for clarity?

Reply:
dda7b33142e2087fa88ef5ef04fefb1334ea72e5